examples/typescript/evm/createSwap.ts

```diff
@@ -0,0 +1,66 @@
+// Usage: pnpm tsx evm/createSwap.ts
+
+import { CdpClient } from "@coinbase/cdp-sdk";
+import { parseEther } from "viem";
+import "dotenv/config";
+
+const cdp = new CdpClient();
+
+async function main() {
```

Comment:
For this we'd want an E2E example of how to sign the returned swap and broadcast using viem